healthcheck/healthcheck.js

```diff
@@ -1,7 +1,13 @@
 const constants = require('../utils/constants')
 
-const getHealthStatus = () => {
-  return constants.CONSTANTS.HEALTHCHECK;
+const getHealthStatus = (req) => {
+  const port = +req.headers.host.split(':')[1];
```

Comment:
This code seems will not work properly on hosts with DNS (for example your https://magic-marks-api.herokuapp.com/) because the 'host' value have not ':' character.

Reply:
fixed